index.html

```diff
@@ -7,104 +7,175 @@
-        <img src="images/Img Placeholder.png" alt="">
-        <h1>Multi-Post Stories</h1>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
-            standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-        <ul class="container-list">
-            <li>css</li>
-            <li>html</li>
-            <li>bootstrap</li>
-            <li>Ruby</li>
-        </ul>
-        <button class="see-project-btn">See Project</button>
-    </section>
-    <section class="project-container">
-        <div class="project">
-            <h2>Profesional Art Printing Data</h2>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
-                industry's standard</p>
-            <ul>
-                <li>html</li>
-                <li>bootstrap</li>
-                <li>Ruby</li>
+    <section class="top-content-container">
+        <div>
+            <ul class="social-media">
+                <li><a href="#"><img src="images/github.png" alt=""></a></li>
+                <li><a href="#"><img src="images/linkedin.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector.png" style="margin-top: 12px;" alt=""></a></li>
+                <li><a href="#"><img src="images/twitter.png" alt=""></a></li>
+                <li><a href="#"><img src="images/medium.png" alt=""></a></li>
             </ul>
-            <button>See Project</button>
         </div>
+        <div class="hey-there">
+            <h1>Hey There. <br>I’m Hasenat</h1>
+            <h2>I’m a Software Developer</h2>
+            <p>I can help you build a product , feature or website. Look <br> through some of my work and experience! If
+                you
+                like what <br> you see and have a project you need coded, <br>
+                don’t hesitate to contact me.</p>
+        </div>
+        <div class="corner-img"></div>
+        <!-- <img class="corner-img" src="images/Illustration-header.png" alt=""> -->
     </section>
-    <section class="project-container">
-        <div class="project">
-            <h2>Profesional Art Printing Data</h2>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
-                industry's standard</p>
-            <ul>
-                <li>html</li>
-                <li>bootstrap</li>
-                <li>Ruby</li>
-            </ul>
-            <button>See Project</button>
+    <section>
+
+        <div class="my-recent-work">
+            <div style="flex-grow: 1;"></div>
+            <div class="recent-work-content">
+                <h2>My Recent Work</h2>
+                <hr>
+            </div>
+            <div style="flex-grow: 1;"></div>
         </div>
+        <section class="multi-post-stories">
+            <img src="images/Img Placeholder.png" alt="">
+            <div>
```

Comment:
- [ ] [OPTIONAL] At this section, the responsiveness is being affected by the `multi-post img` 😞 . Kindly ensure that the image has flexible `width` which will adapt to different screen sizes. You may consider using this [W3schools Tutorial](https://www.w3schools.com/css/css_rwd_images.asp) for clear clarification on how to do it 👍 .

Your design 😎 
![multi](https://user-images.githubusercontent.com/99394374/213858955-1f368bba-4b8e-426e-b575-d867f2412d3f.PNG)